4-WebApp-your-API/4-1-MyOrg/AppCreationScripts-withCert/Configure.ps1

```diff
@@ -262,10 +277,13 @@ Function ConfigureApplications
                                                           RedirectUris = "https://localhost:44321/", "https://localhost:44321/signin-oidc"; `
```

Comment:
Do we really need "https://localhost:44321/" as a redirect URI? I would not think so?

Reply:
removed

Reply:
removed